src/main/java/usf/edu/cutr/grha/model/LocationFixData.kt

```diff
@@ -0,0 +1,13 @@
+package usf.edu.cutr.grha.model
+
+data class LocationFixData
+(
+val fix: String,
```

Comment:
You don't need the `fix` field here, assuming this comes from the GPSTest log. That field just designates that a record is a location fix and that the rest of the fields in that record represent data that should be stored in this class.